mobile/status.go

```diff
@@ -1394,25 +1686,42 @@ func ToChecksumAddress(address string) string {
 }
 
 func DeserializeAndCompressKey(DesktopKey string) string {
+	return logAndCallString(deserializeAndCompressKey, DesktopKey)
+}
+
+func deserializeAndCompressKey(DesktopKey string) string {
 	deserialisedKey := MultiformatDeserializePublicKey(DesktopKey, "f")
 	sanitisedKey := "0x" + deserialisedKey[5:]
 	return CompressPublicKey(sanitisedKey)
 }
 
+type InitLoggingRequest struct {
+	logutils.LogSettings
+	LogRequestGo   bool
+	LogRequestFile string
```

Comment:
`json` tags missing?

Reply:
Just better to explicitly define them. Otherwise it's too simple to rename the field and forget to thing about clients.